lib/sen_tweet_web/controllers/stream_controller.ex

```diff
@@ -4,4 +4,19 @@ defmodule SenTweetWeb.StreamController do
   def index(conn, _params) do
     render(conn, "index.html", streams: Bitfeels.all_streams())
   end
+
+  def new(conn, _params) do
+    render(conn, "new.html")
+  end
+
+  def create(conn, %{"user" => user, "track" => track, "filter_level" => filter_level}) do
+    Bitfeels.twitter_stream(user, track, filter_level)
+    redirect(conn, to: "/admin/streams")
+  end
+
+  def delete(conn, %{"id" => id}) do
+    [user, track] = String.split(id, "_")
+    Bitfeels.stop_stream(user, track)
```

Comment:
When trying to match on `%{"id" => user <> "_" <> track}` I was getting the following exception: 
```
== Compilation error in file lib/sen_tweet_web/controllers/stream_controller.ex ==
** (ArgumentError) the left argument of <> operator inside a match should be always a literal binary as its size can't be verified, got: user
    (elixir) lib/kernel.ex:1803: Kernel.invalid_concat_left_argument_error/1
```
`String.split` works fine but isn't as sexy. Maybe you know how to resolve?

Reply:
Ahh yeah, this match isn't going to work. There can only an unknown length at the end of the match. Here the length of the beginning _and_ end of the match are unknown.

For example, this match works
```
👉  id = "bitfeels_bitcoin"
"bitfeels_bitcoin"
👉  "bitfeels_" <> track = id
"bitfeels_bitcoin"
```

If user was a fixed length we could apply a different method here. Since user is an arbitrary length, split is a good choice here.